DSharpPlus/Entities/Application/DiscordApplication.cs

```diff
@@ -78,17 +78,157 @@ public sealed class DiscordApplication : DiscordMessageApplication, IEquatable<D
     /// Gets the team which owns this application.
     /// </summary>
     public DiscordTeam? Team { get; internal set; }
+    
+    /// <summary>
+    /// Public key used to verify http interactions
+    /// </summary>
+    public string VerificationKey { get; internal set; }

```

Comment:
nit(consistency): This is VerificationKey, but on the transport object it's VerifyKey (which lines up with the field name)